model_info/trwiki.goodfaith.md

```diff
@@ -15,67 +15,67 @@ Model Information:
 	 - python_implementation: 'CPython'
 	 - python_revision: ''
 	 - python_version: '3.5.3'
-	 - release: '4.9.0-8-amd64'
+	 - release: '4.9.0-9-amd64'
 	
 	Statistics:
-	counts (n=33495):
+	counts (n=19361):
```

Comment:
counts changed quite a bit here

Reply:
Hmm.  From what I can tell, it looks like we fixed a bug.  The big question I have is (1) how did we have such absurd counts in the past and (2) what effect did this have on predictions?  We should probably increment the version number of the model so that everything rights itself downstream.

Reply:
I looked through the file history here and nothing is obvious.  I'm guessing that this was an issue with a corrupt local file when training.  The fact that the # of target observations doesn't change probably reflects the underlying issue.  I still don't quite understand it but it looks like a good improvement in the pipeline to fix it here.